config/libc/newlib.in.2

```diff
@@ -65,6 +65,77 @@ config LIBC_NEWLIB_NANO_FORMATTED_IO
       with size constraint issues.  This option does not affect wide-char
       formatted I/O functions.
 
+config LIBC_NEWLIB_REGISTER_FINI
+    bool
+    prompt "Enable Register finalization"
+    depends on LIBC_NEWLIB_2_2_or_later
+    help
+      Enable finalization function registration using atexit.
```

Comment:
So, is this "finalizing registers" or "registering a finalization function"? Please correct either the prompt, or the help message.